### PR TITLE
Adding explorer v1.104

### PR DESCRIPTION
**Pull request by gobinathm**
- Including Explorer v1.104

*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/explorer.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  version '1.104'
+  sha256 'fcf3ec43ab2dff7f9e734192d3d327eebce80240007c4848b433f5d0f6ae8605'
+
+  url "https://github.com/jfbouzereau/explorer/releases/download/#{version}/Explorer-darwin-x64.zip"
+  appcast 'https://github.com/jfbouzereau/explorer/releases.atom',
+          checkpoint: '72093729e37cd133de303b4f9748c20453d1188ba8763eaf70b9d3b5fe92319d'
+  name 'explorer'
+  homepage 'https://github.com/jfbouzereau/explorer'
+
+  app 'explorer'
```
Comment (vitorgalvao) [inline Casks/explorer.rb]: `Explorer-darwin-x64.app`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/explorer.rb (hunk 1 above)
Comment (gobinathm) [inline Casks/explorer.rb]: style check / audit / installation did not fail with that name :( hence i ended up leaving that.  Pushing an update..

Comment (gobinathm) [inline Casks/explorer.rb]: @vitorgalvao pushed an update

Comment (reitermarkus) [inline Casks/explorer.rb]: > installation did not fail with that name

But did it actually install what you expected?

Comment (gobinathm) [inline Casks/explorer.rb]: @reitermarkus , actually i was wrong, in my machine it did not fail due to a problem in my local. Tried in another machine it did fail.

File: Casks/explorer.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cask 'explorer' do
+  version '1.104'
+  sha256 'fcf3ec43ab2dff7f9e734192d3d327eebce80240007c4848b433f5d0f6ae8605'
+
+  url "https://github.com/jfbouzereau/explorer/releases/download/#{version}/Explorer-darwin-x64.zip"
+  appcast 'https://github.com/jfbouzereau/explorer/releases.atom',
+          checkpoint: '72093729e37cd133de303b4f9748c20453d1188ba8763eaf70b9d3b5fe92319d'
+  name 'explorer'
```
Comment (miccal) [inline Casks/explorer.rb]: Please change this to `Explorer`.

**Review by miccal — CHANGES_REQUESTED** [review]

File: Casks/explorer.rb (hunk 2 above)
Comment (gobinathm) [inline Casks/explorer.rb]: @miccal Its updated.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**